include/seqan3/core/algorithm/all.hpp

```diff
@@ -73,7 +73,7 @@
  * \include test/snippet/core/algorithm/configuration_get_or.cpp
  */
 #include <seqan3/core/algorithm/algorithm_result_generator_range.hpp>
-#include <seqan3/core/algorithm/bound.hpp>
+#include <seqan3/core/algorithm/bound.hpp> // ToDo: remove this with the release SeqAn-3.1.0.
```

Comment:
please add the macros to issue a warning message when the versions have been updated so we can't forget about this.